www/index.rst

```diff
@@ -28,24 +29,48 @@ few restrictions.
 Getting Started
 ---------------
 
-- `Getting NumPy <http://www.scipy.org/scipylib/download.html>`__
-- `Installing the SciPy Stack <http://www.scipy.org/install.html>`__
-- `NumPy and SciPy documentation page <http://docs.scipy.org/doc/>`__
-- `NumPy Tutorial <https://docs.scipy.org/doc/numpy/user/quickstart.html>`__
-- `NumPy for MATLAB© Users <https://docs.scipy.org/doc/numpy/user/numpy-for-matlab-users.html>`__
-- `NumPy functions by category <https://docs.scipy.org/doc/numpy/reference/routines.html>`__
-- `NumPy Mailing List <http://www.scipy.org/scipylib/mailing-lists.html>`__
+- :ref:`obtaining`
+- `NumPy Tutorial <devdocs/user/quickstart.html>`__
```

Comment:
Can we keep these `devdocs` links as absolute links instead? The devdocs aren't part of this repo, and as relative links they're impossible to verify before deployment with a local build of the site.

Reply:
ok